PlayCover/Model/PlayApp.swift

```diff
@@ -38,6 +39,10 @@ class PlayApp: BaseApp {
             if prohibitedToPlay {
                 await clearAllCache()
                 throw PlayCoverError.appProhibited
+            } else if hasMacVersion, await !runMacOSWarning() {
+                await clearAllCache()
```

Comment:
you don't need to clear the cache here, it will remove user data if they just wanted to check it your provided data out